matcher/cache.go

```diff
@@ -35,9 +35,12 @@ type Cache interface {
 	// between [fromNanos, toNanos).
 	ForwardMatch(namespace, id []byte, fromNanos, toNanos int64) rules.MatchResult
 
-	// Register sets the result source for a given namespace.
+	// Register sets the source for a given namespace.
 	Register(namespace []byte, source Source)
 
+	// Update updates the source for a given namespace.
```

Comment:
Update the comment here to something inline of "clean up the cached results for the old rules", since this method does not actually update the source?

Reply:
Would `Refresh` be a better name?

Reply:
same